plugins/generator-1.15.2/forge-1.15.2/templates/json/tag.json.ftl

```diff
@@ -15,6 +15,10 @@
             "${generator.getResourceLocationForModElement(value)}"
             <#if value?has_next>,</#if>
           </#list>
+      <#elseif data.type == "Entities">
+          <#list data.entities as value>
+            "${generator.map(value, "entities", 2)}"<#if value?has_next>,</#if>

```

Comment:
Only do this when it is not CUSTOM: entity, for custom entites, use previous approach

Reply:
How can I know when `value` is CUSTOM:entity?